WordPress/Classes/ViewRelated/NUX/SignupUsernameTableViewController.swift

```diff
@@ -4,7 +4,7 @@ import WordPressAuthenticator
 class SignupUsernameTableViewController: UITableViewController, SearchTableViewCellDelegate {
     open var currentUsername: String?
     open var displayName: String?
-    open var delegate: SignupUsernameViewControllerDelegate?
+    open weak var delegate: SignupUsernameViewControllerDelegate?
```

Comment:
This `delegate` is used in a similar way as https://github.com/wordpress-mobile/WordPress-iOS/pull/21064#discussion_r1260368371

https://github.com/wordpress-mobile/WordPress-iOS/blob/48b8a391394f2b4aee9b886abe21f62a521c0a81/WordPress/Classes/ViewRelated/NUX/SignupEpilogueViewController.swift#L80-L88